files/default/neutron-ha-tool.py

```diff
@@ -222,13 +227,15 @@ def run(args):
     elif args.l3_agent_migrate:
         LOG.info("Performing L3 Agent Migration for Offline L3 Agents")
         errors = retry_with_backoff(l3_agent_migrate, args)(
-            qclient, args.noop, args.now, args.wait_for_ports)
+            qclient, args.noop, args.now, args.wait_for_ports,
+            args.ssh_delete_namespace)
```

Comment:
Yeah this is strengthening my suggestion in #7 that it's worth just passing `args` instead of all these params:

* https://sourcemaking.com/refactoring/smells/long-parameter-list